README.md

```diff
@@ -216,7 +216,17 @@ link_design top
 write_db reg1.db
 ```
 
-#### Example Scripts
+#### Run in Python
+
+OpenROAD can be run in python mode as following
```

Comment:
I think "with a python command interpreter" is more appropriate than "python mode".